docker/Dockerfile.linux

```diff
@@ -0,0 +1,13 @@
+FROM node:buster
```

Comment:
Note that the binaries are not going to work with any distro older than Debian Buster. 

Per previous discussions, use RHEL toolchain instead unless you have a better idea.

Reply:
I have no experience in this, could you fix it?

Reply:
Next week, as I said.